workflow/scripts/krakenuniq2krona.R

```diff
@@ -1,16 +1,16 @@
 #This is a script for visualizing filtered KrakenUniq output with Krona. Run this script as:
-#Rscipt krakenuniq2krona.R bal004.krakenuniq.output bal004.sequences.krakenuniq
+#Rscipt krakenuniq2krona.R bal004.krakenuniq.output bal004.sequences.krakenuniq n_unique_kmers n_tax_reads
 
 args = commandArgs(trailingOnly=TRUE)
 
 #Read and filter KrakenUniq output
 df<-read.delim(args[1],comment.char="#",header=TRUE)
 colnames(df)[1]<-"Pers_Reads"
 print(paste0("Original data set dimensions: ",dim(df)[1]," and ",dim(df)[2]))
-df<-df[df$kmers>1,]
+df<-df[df$kmers>args[3],]
```

Comment:
See comment above, integers args should be converted.

The other option is removing this filtering and having `krakenuniq2krona.R` read the output as filtered by `filter_krakenuniq.R` if the two are always run in sequence, and remove this filtering altogether from this